<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="exportTabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="roi_tab">
      <attribute name="title">
       <string>ROIs</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="manualGroup">
         <property name="title">
          <string>ROI Table</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="ROITableWidget" name="table_view">
            <attribute name="horizontalHeaderHighlightSections">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="removeBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Removes the selected row from the table.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="addBtn">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Adds a new row to the table.&lt;/p&gt;&lt;p&gt;Slice index defaults to the current preview slice index.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exportButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Export spectrum</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="image_tab">
      <attribute name="title">
       <string>Image</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Output Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="image_output_mode_combobox">
         <item>
          <property name="text">
           <string>Single Spectrum</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2D Binned</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Error Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="transmission_error_mode_combobox">
         <item>
          <property name="text">
           <string>Standard Deviation</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Propagated</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="bin_size_label">
         <property name="text">
          <string>Bin Size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="bin_size_spinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="bin_step_label">
         <property name="text">
          <string>Bin Step</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="bin_step_spinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="exportButtonRITS">
         <property name="text">
          <string>Export to RITS</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="ROIPropertiesTableWidget" name="roi_properties_widget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ROITableWidget</class>
   <extends>QTableView</extends>
   <header>mantidimaging.gui.windows.spectrum_viewer.view</header>
  </customwidget>
  <customwidget>
   <class>ROIPropertiesTableWidget</class>
   <extends>QWidget</extends>
   <header>mantidimaging.gui.windows.spectrum_viewer.view</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
